_docs/deprecated_and_archived_modules.md

```diff
@@ -17,7 +17,7 @@ The decision to archive a module isn't necessarily final and can be undone.
 
 ### Deprecation process
 
-* An issue needs to be raised on the module to discuss whether it should be archived or not. The decision will be made by lazy consensus (as described in our governance guidelines)
+* An issue needs to be raised on the module (or a post to our mailing list) to discuss whether it should be archived or not. The decision will be made by lazy consensus (as described in our governance guidelines)
```

Comment:
I'd prefer to keep the issue mandatory because even for archived modules you can view the issues.
```suggestion
* An issue needs to be raised on the module to discuss whether it should be archived or not. An additional post to the mailing list for better visibility is recommended. The decision will be made by lazy consensus (as described in our governance guidelines)
```